benchmark/benchmark_util/data_table_benchmark_util.cpp

```diff
@@ -73,15 +73,15 @@ LargeDataTableBenchmarkObject::LargeDataTableBenchmarkObject(const std::vector<u
                                                              storage::RecordBufferSegmentPool *buffer_pool,
                                                              std::default_random_engine *generator, bool gc_on,
                                                              storage::LogManager *log_manager)
-    : txn_length_(txn_length),
-      operation_ratio_(std::move(operation_ratio)),
-      generator_(generator),
-      layout_({attr_sizes}),
+    : layout_({attr_sizes}),
```

Comment:
Why was this class changed?

Reply:
Check-clang-tidy would like this order since there is padding in the data_table